SensusService/Protocol.cs

```diff
@@ -570,7 +584,97 @@ public string Description
             }
         }
 
-        [EntryIntegerUiProperty("Participation Horizon (Days):", true, 16)]
+        [DateUiProperty("Start Date:", true, 16)]
+        public DateTime StartDate
+        {
+            get
+            {
+                return _startTime;
+            }
+            set
+            {
+                _startTime = new DateTime(value.Year, value.Month, value.Day, _startTime.Hour, _startTime.Minute, _startTime.Second);
```

Comment:
Nullable `_startTime` will bite you here. Why can't we have a single `StartDateTime` that captures everything in one property? The dependency between `StartDate` and `StartTime` is awkward.